packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -51,7 +51,7 @@ - (instancetype)initWithFrame:(CGRect)frame
     NSDictionary<NSString*, id>* settings = args[@"settings"];
     [self applySettings:settings];
     NSString* initialUrl = args[@"initialUrl"];
-    if (initialUrl && initialUrl != [NSNull null]) {
+    if (initialUrl && ![initialUrl isKindOfClass:[NSNull class]]) {
```

Comment:
Curious, should we prefer this form over the former?

Reply:
Yeah, so checking the class by isKindOfClass covers the case to check if initialUrl is [NSNull null]; and using isKIndOfClass will suppress the warning for type not matching

Reply:
Ohh good to know, thanks! I missed the type warning.
cc @cbracken